packages/website/scripts/compileDocsForLLM.mjs

```diff
@@ -19,20 +54,22 @@ This file is auto-generated from the documentation files in the Squiggle reposit
     return readFile(grammarFiles[0]);
   };
 
-  const getDocumentationContent = async () => {
+  const getGuideContent = async () => {
     const documentationFiles = await glob(
-      "./src/pages/docs/{Api,Guides}/*.{md,mdx}"
+      "./src/pages/docs/{Guides}/*.{md,mdx}"
```

Comment:
Are all API files in templates yet? I thought there's some stuff that's still not converted, https://github.com/quantified-uncertainty/squiggle/tree/1ff759c7f7aa015dbe61d638676053fe140b5ebc/packages/website/src/pages/docs/Api